practice/1_python_part_1/task3.py

```diff
@@ -16,4 +16,22 @@
 
 
 def build_from_unique_words(*lines: Iterable[str], word_number: int) -> str:
-    ...
+  words = []
+  final = ''
+
+  for line in lines:
+    unique =[]
+    temp = line.split()
+
+    for i in temp:
+      if i not in unique:
+        unique.append(i)  
+    if word_number > len(unique)-1:
+      pass
+    else:
+      words.append(unique[word_number])
+
+  for word in words:
+    final = final + word +' '
+  
+  return final[:-1]
```

Comment:
Try to use `str#join` method to avoid this `final[:-1]`. Moreover, if your strings will be very long, `join` will show better performance than concatenation (e.g. https://towardsdatascience.com/do-not-use-to-join-strings-in-python-f89908307273)